doc/api/deprecations.md

```diff
@@ -634,6 +634,17 @@ Type: Runtime
 
 *Note*: change was made while `async_hooks` was an experimental API.
 
+<a id="DEP00XX"></a>
```

Comment:
I don’t care much but I think @jasnell has been telling people multiple times to assign the number while landing :)